test/customer_parser_test.rb

```diff
@@ -4,6 +4,6 @@
 
 class CustomerParserTest < MiniTest::Test
   def test_it_exists
-    assert CustomerParser.new('test/support/sample_customers.csv')
+    assert CustomerParser.new('test/support/customers.csv')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.